CHANGES

```diff
@@ -1,5 +1,19 @@
 Version 2.0
 ===========
+Backwards Incompatible Changes
+------------------------------
+* Any installed ZenPacks using older versions of zenpacklib.py will continue to function unchanged.
```

Comment:
_Backwards Incompatible Changes_ can probably just be made regular text like _Features_ and _Fixes under each release as lists immediately after section titles don't look very good. You'd normally want a regular paragraph following a section title.

There needs to be a blank line before beginning a list in reStructuredText.